tests/unit/models/builder-test.js

```diff
@@ -498,4 +502,42 @@ describe('models/builder.js', function() {
       });
     });
   });
+
+  if (isExperimentEnabled('BROCCOLI_2')) {
+    describe('fallback from broccoli 2 to broccoli-builder', function() {
+      it('falls back to broccoli-builder if an InvalidNode error is thrown for read/rebuild api', function() {
+        let spy = sinon.spy(console, 'warn');
```

Comment:
This should be removed once the tests properly instantiate the builder with our MockUI object. You can reference other tests to see how to make assertions on the output when using it.